### PR TITLE
fix(chat): add optimistic update for new entity creation (Issue #725)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
- Issue: https://github.com/epam/ai-dial-chat/issues/725

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-741.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-741.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts [hunk 1]
```diff
@@ -28,8 +28,8 @@ export class ConversationApiStorage extends ApiEntityStorage<
 > {
   mergeGetResult(info: ConversationInfo, entity: Conversation): Conversation {
     return {
-      ...entity,
       ...info,
+      ...entity,
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts:32]: Please, revert it back

File: apps/chat/src/utils/app/data/storages/api/prompt-api-storage.ts [hunk 2]
```diff
@@ -12,8 +12,8 @@ import { ApiEntityStorage } from './api-entity-storage';
 export class PromptApiStorage extends ApiEntityStorage<PromptInfo, Prompt> {
   mergeGetResult(info: Entity, entity: Prompt): Prompt {
     return {
-      ...entity,
       ...info,
+      ...entity,
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/data/storages/api/prompt-api-storage.ts:16]: Please, revert it back

**Review by IlyaBondar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**